indigo/distribution.yaml

```diff
@@ -7270,6 +7270,11 @@ repositories:
       url: https://bitbucket.org/crl/multisense_ros
       version: default
     status: developed
+  muse_bldc_motor_drive:
+    doc:
+      type: git
+      url: https://bitbucket.org/muserobotics/muse_ros_package
```

Comment:
Please use the full url here to pass CI `https://bitbucket.org/muserobotics/muse_ros_package.git`